build_WOFF.py

```diff
@@ -0,0 +1,19 @@
+import sys
+from pathlib import Path
+
+from fontTools import ttLib
+
+for file in Path("build").glob("*.ttf"):
+    ttfile = ttLib.TTFont(file)
+
+    if "TSI0" in ttfile:
```

Comment:
Shouldn't this exit? It is an error after all and there will be no WOFF2 file, even though the next print statement says "all done".

Reply:
I get super annoyed when scripts that are supposed to run on multiple files exits out after the first one. What if there’s more files that the error would occur on? I’d prefer to know it all at once rather than only see one at a time.

But I do agree that the strings could be better. :)